baselines/driving_smarts/example.py

```diff
@@ -0,0 +1,25 @@
+import gymnasium as gym
+
+from smarts.env.gymnasium.wrappers.episode_limit import EpisodeLimit
+from smarts.env.gymnasium.wrappers.metrics import CompetitionMetrics
+
+env = gym.make(
+    "smarts.env:driving-smarts-competition-v0",
+    scenario="3lane_merge_multi_agent",
+    sumo_headless=False,
+)
+env = CompetitionMetrics(env)
+env = EpisodeLimit(env, 5)
+
+observation, info = env.reset(seed=42)
+while not info.get("reached_episode_limit"):
+    # Generally [x-coordinate, y-coordinate, heading]
+    action = env.action_space.sample()
+    observation, reward, terminated, truncated, info = env.step(action)
+    env.observation_space.contains(observation)
+
+    if terminated or truncated:
+        observation, info = env.reset()
```

Comment:
Can we avoid introducing `EpisodeLimit` env wrapper by directly counting the number of `env.reset()` called?

Reply:
I have removed the wrapper because it does not make for an improvement and causes confusion compared to `TimeLimit` as you mentioned elsewhere.